configs/qq_weixin_pay.json

```diff
@@ -1,13 +1,10 @@
 {
   "index_name": "qq_weixin_pay",
   "start_urls": [
-    "https://pay.weixin.qq.com/wiki/doc/apiv3/wxpay/pages/index.shtml",
-    "https://pay.weixin.qq.com/wiki/doc/apiv3/wxpay/",
-    "https://pay.weixin.qq.com/wiki/doc/api/index.html",
-    "https://pay.weixin.qq.com/wiki/doc/api/"
+    "https://pay.weixin.qq.com/wiki/doc/apiv3/open/",
   ],
   "stop_urls": [
-    "\\?"
+    "https://pay.weixin.qq.com/wiki/doc/apiv3/open/"
```

Comment:
Hi,

```suggestion
```

You are using the same URL for the `start_urls` and the `stop_urls`, if the idea behind is to avoid scrapping the `start_urls`, please use `scrape_start_urls: false`